x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/analysis/CancellationTests.java

```diff
@@ -51,12 +55,13 @@ public void testCancellationBeforeFieldCaps() throws InterruptedException {
         Client client = mock(Client.class);
         EqlSearchTask task = mock(EqlSearchTask.class);
         when(task.isCancelled()).thenReturn(true);
+        ClusterService mockClusterService = mockClusterService();
 
         IndexResolver indexResolver = new IndexResolver(client, randomAlphaOfLength(10), DefaultDataTypeRegistry.INSTANCE);
         PlanExecutor planExecutor = new PlanExecutor(client, indexResolver, new NamedWriteableRegistry(Collections.emptyList()));
         CountDownLatch countDownLatch = new CountDownLatch(1);
-        TransportEqlSearchAction.operation(planExecutor, task, new EqlSearchRequest().query("foo where blah"), "", "", "node_id",
-            new ActionListener<>() {
+        TransportEqlSearchAction.operation(planExecutor, task, new EqlSearchRequest().query("foo where blah"), "",
+            mock(TransportService.class), mockClusterService, new ActionListener<>() {
```

Comment:
minor: since `mock(TransportService.class)` is reused, you could also assign it to a var.